tiledb/sm/query/readers/dense_reader.cc

```diff
@@ -386,7 +386,9 @@ Status DenseReader::dense_read() {
     if (condition_names.count(name) == 0) {
       names.emplace_back(name);
     }
+  }
 
+  for (auto& name : names) {
     if (array_schema_.var_size(name)) {
       var_names.emplace_back(name);
     } else {
```

Comment:
fixed_names is not used... We can remove.

Reply:
Done.